.github/workflows/test-environment.yml

```diff
@@ -135,15 +135,15 @@ jobs:
       TF_VAR_ess_region: ${{ inputs.ess-region }}
       DEPLOYMENT_NAME: ${{ inputs.deployment_name }}
       TF_VAR_serverless_mode: ${{ inputs.serverless_mode }}
-      TEST_AGENTLESS: ${{ inputs.serverless_mode }}
```

Comment:
I could be missing something, but this seems to be committed by mistake. We should keep it conditional based on the workflow input value.

Reply:
We want to deploy agentless for both ESS and Serverless now. I left the environment variable there in case we disable it a briefly (bugs,  changes, etc).